pkg/kudoctl/cmd/init/crds.go

```diff
@@ -268,6 +195,10 @@ func generateCrd(kind string, plural string) *apiextv1beta1.CustomResourceDefini
 			Conditions:     []apiextv1beta1.CustomResourceDefinitionCondition{},
 			StoredVersions: []string{},
 		},
+		TypeMeta: metav1.TypeMeta{
```

Comment:
I.e., only `InstanceCrd` will have `TypeMeta` set, `operatorCrd` and `operatorVersionCrd` won't. What are the consequences of this, given that the existing code made sure that `TypeMeta` is set in `CRDs` but not in `installCrds`?

Reply:
hmm why not? I moved the code to generateCrd and all the methods are using it.

Reply:
Oh! You're right, sorry. Github truncated the diff in a misleading place, I thought this is `generateInstance` while we're in fact in `generateCrd` here.